Move runtime IP to EditorPref / DevAuthToken to PlayerPref (#961)
spatialos/gdk-for-unity: CHANGELOG.md

### Changed

- Upgraded the project to be compatible with `2019.1.3f1`.

> - Moved Runtime IP from the `GdkToolsConfiguration.json` to an Editor Preference.

Reviewer: ```suggestion
- Moved Runtime IP from the `GdkToolsConfiguration.json` to the Editor Preferences.
```

> - Moved Dev Auth Token to a PlayerPref.

Reviewer: ```suggestion
- Moved Dev Auth Token to the Player Preferences.
```

>     - Added a setting in `GdkToolsConfiguration` to let users configure whether a DevAuthToken.txt should be generated or not.

Reviewer: ```suggestion
    - Added a setting in `GdkToolsConfiguration` to let users configure whether a `DevAuthToken.txt` should be generated or not.
```
